vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -1164,6 +1167,8 @@ public UpdateQueueData getUpdateQueueData() {
 
     private final List<GridSortOrder<T>> sortOrder = new ArrayList<>();
 
+    private SerializableConsumer<?> filterSlot;
```

Comment:
Maybe add a comment that this is used to only update the filter from the listdataview

Reply:
Added